app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/stats/AppTrackerBlockingStatsRepository.kt

```diff
@@ -106,15 +99,9 @@ class RealAppTrackerBlockingStatsRepository @Inject constructor(
 ) : AppTrackerBlockingStatsRepository {
```

Comment:
`data class DataTransfer(
    val dataSize: Long = 0,
    val numberPackets: Long = 0
)`

seems not needed anymore.

Reply:
Awesome, will remove